src/pages/edit-profile/edit-profile.tsx

```diff
@@ -0,0 +1,114 @@
+import { Box, TextField, Container, Typography, Button } from '../../shared/ui/elements/components';
```

Comment:
where is React?

use shared as absolute pathes

Reply:
I need to import React in all files? I thought it automatically already exists

Reply:
Yes, explicitly is better than implicitly (python zen) :)